src/test/java/org/adoptopenjdk/maven/plugins/JsplitpkgscanMojoTest.java

```diff
@@ -0,0 +1,115 @@
+package org.adoptopenjdk.maven.plugins;
+
+import org.apache.maven.artifact.Artifact;
+import org.apache.maven.model.Dependency;
+import org.apache.maven.plugin.testing.AbstractMojoTestCase;
+import org.apache.maven.plugin.testing.stubs.ArtifactStub;
+import org.mockito.ArgumentMatcher;
+
+import static org.mockito.Mockito.*;
+
+import javax.tools.Tool;
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.Set;
+
+public class JsplitpkgscanMojoTest extends AbstractMojoTestCase {
+    private Tool tool;
+    private JsplitpkgscanMojo jsplitpkgscanMojo;
+
+    /**
+     * {@inheritDoc}
+     */
+    protected void setUp()
+            throws Exception {
+        // required
```

Comment:
What is required and why?

Reply:
Added an override annotation and removed the comment-